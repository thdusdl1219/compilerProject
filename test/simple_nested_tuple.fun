fun main(argc:int):int =
printint(#1 #0 <<1,2,3>,<2,3,4>,<<<1,2,3,4>,<2,3,4>>,1,2,3>>);
printint(#0 #1 <<1,2,3>,<2,3,4>,<<<1,2,3,4>,<2,3,4>>,1,2,3>>);
printint(#2 #2 <<1,2,3>,<2,3,4>,<<<1,2,3,4>,<2,3,4>>,1,2,3>>);
printint(#1 #0 #0 #2 <<1,2,3>,<2,3,4>,<<<1,2,3,4>,<2,3,4>>,1,2,3>>);
printint(#0 #1 #0 #2 <<1,2,3>,<2,3,4>,<<<1,2,3,4>,<2,3,4>>,1,2,3>>);
printint(1414);0
