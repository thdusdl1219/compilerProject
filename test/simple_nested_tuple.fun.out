[2, 2, 2, 2, 2, 1414, 0]
